### PR TITLE
Issue 139: MLP multi-class classifier

### LINKED ISSUE #139: Add Multilayer Perceptron (MLP) primitive for multi-class softmax classification
Related to #121

### PR DESCRIPTION
**Pull request by Hector-hedb12**
Resolves #139

### CODE REVIEW

**Comment by Hector-hedb12** [comment]
@csala as `softmax` is just a value in the `last_dense_activation` parameter, I was thinking in change the name of this primitive from `keras.Sequential.MLPSoftmaxClassifier.json` to `keras.Sequential.MLPClassifier.json` or `keras.Sequential.MLPMultiClassClassifier.json`. What do you think?

**Comment by csala** [comment]
@Hector-hedb12 I think that we can keep them separated, as there are a couple of other differences, and they clearly respond to different needs (binary/multi-class).

What I would do, though, is renaming this to MLPMultiClassClassifier, so we can have Binary/MultiClass.

File: pipelines/keras.Sequential.MLPMultiClassClassifier.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        }
+    },
+    "output_names": {
+        "mlprimitives.custom.counters.UniqueCounter#1": {
+            "counts": "classes"
+        }
+    },
+    "init_params": {
+        "keras.Sequential.MLPMultiClassClassifier#1": {
+            "epochs": 1,
```
Comment (csala) [inline pipelines/keras.Sequential.MLPMultiClassClassifier.json]: Since this example is very quick, let's remove this and let the default (20) be used.

File: mlprimitives/jsons/keras.Sequential.MLPMultiClassClassifier.json [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        "fixed": {
+            "classification": {
+                "type": "bool",
+                "default": true
+            },
+            "dense_units": {
+                "type": "int",
+                "default": 64
+            },
+            "input_dim": {
```
[the remark this answers is not in the document]
Comment (csala) [reply mlprimitives/jsons/keras.Sequential.MLPMultiClassClassifier.json]: When setting this as a `fit` argument, I would call it `features` (see #147), and add a description explaining that this is the number of features in X

**Comment by Hector-hedb12** [comment]
Updated @csala

**Comment by csala** [comment]
Looks good @Hector-hedb12 !
